File: c#/src/WOT.TAK.Connection/src/PlainConnector.cs
namespace WOT.TAK.Connection
{
    internal class PlainConnector : TAKServerConnector
    {
        private string serverUrl;
        private string serverPort;
        private string cotResponsesDirPath;
        private SocketFactory socketFactory;

        public PlainConnector(string serverUrl, string serverPort, string cotResponsesDirPath, SocketFactory socketFactory)
        {
            this.serverUrl = serverUrl;
            this.serverPort = serverPort;
            this.cotResponsesDirPath = cotResponsesDirPath;
            this.socketFactory = socketFactory;
        }

        public void Close()
        {
            throw new NotImplementedException();
        }

        public void Connect()
        {
            throw new NotImplementedException();
        }

        public void SendFile(string path)
        {
            throw new NotImplementedException();
        }
    }
}

File: connection/src/main/java/org/wot/tak/connection/PlainConnector.java
package org.wot.tak.connection;

import java.io.IOException;

public class PlainConnector extends TCPConnector{

    protected PlainConnector(String url, String port, String responseStoragePath, SocketFactory sFactory) {
        super(url, port, responseStoragePath, sFactory);
    }

    public void connect() throws IOException {
        s = socketFactory.createSocket(url, port);
        super.connect();
    }

}
